common/system/tasks/trigger_manager.cpp

```diff
@@ -82,6 +120,26 @@ bool triggerManager_setTriggerConfig(uint8_t triggerNum,
   return false;
 }
 
+const TriggerState triggerManager_getExpectedTriggers(uint8_t i) {
+  if (i < NUM_PYRO) {
+    return triggerConnectivityStatus[i];
+  }
+  return TriggerState::NOT_CONFIGURED;
+}
+
+/**
+ * Manage array bc Sam says users are stupid
+ */
+const char* triggerManager_triggerStatusToString(TriggerState i) {
+  const char* triggerStatusToString[] = {"CONNECTED", "NOT_CONNECTED",
```

Comment:
This should be static and placed outside of the function.

Reply:
:)